content/influxdb3/core/reference/influxDB3 internals /data durability/_index.md

```diff
@@ -0,0 +1,17 @@
+---
```

Comment:
The file path should be
`content/influxdb3/core/reference/internals/durability/_index.md`

Reply:
Fixed in 7a734f8